package.json

```diff
@@ -28,7 +28,7 @@
   "license": "Apache-2.0",
   "type": "module",
   "engines": {
-    "node": ">=14.0"
+    "node": ">=18.0"
```

Comment:
- [ ] We can do `^20.15.0` as this is the version `yarn@3.8.2` uses.
- [ ] I think it would be wise to update the `@types/node` package then as well as it's currently using version `20.14.5`.